compiler/crates/relay-compiler/src/main.rs

```diff
@@ -45,6 +45,9 @@ struct Opt {
     /// Run the persister even if the query has not changed.
     #[structopt(long)]
     repersist: bool,
+
+    #[structopt(default_value = "0", long)]
+    persist_concurrency: usize,
```

Comment:
I think we should move this to `persist_config`. Adding this to CLI is spreading persist configuration across these configs.

Our general approach is to reduce number of CLI flags as much as possible, and move everything to config files.

Reply:
@alunyov I agree it makes sense to add to `persist_config`. I had trouble with that approach because it doesn't seem like `persist_config` is available [when the RemotePersister is instantiated](https://github.com/robrichard/relay/blob/persist_concurrency/compiler/crates/relay-compiler/src/main.rs#L112) because it's part of the project config. I'm still very much a rust beginner so it's likely I'm missing something.

How would you recommend I approach this?